assets/src/dashboard/app/views/myStories/content/test/content.js

```diff
@@ -124,7 +124,7 @@ describe('My Stories <Content />', function () {
       { features: { enableInProgressStoryActions: false } }
     );
 
-    expect(getByText('Create a story to get started!')).toBeInTheDocument();
+    expect(getByText('Start telling Stories.')).toBeInTheDocument();
```

Comment:
I've been running across tests breaking for simple text updates and feel like it shouldn't happen so often. Maybe it's better practice to add `testid`s vs getting elements by their text? Thoughts?

Don't need to change it here, but curious about ppls thoughts as like a future best practice to start implementing or not

Reply:
good idea, either testids or sourcing text from 1 place, it'd be great to feed strings to pages from json or something. make it easier for text to be updated by non engineering.

Reply:
Ooo yea that'd also be dope